<launch>
<!-- Launch the atlas & vehicle & fire hose world-->
<include file="$(find atlas_utils)/launch/atlas_drc_vehicle_fire_hose.launch" />

<!-- Start Drive Init server -->
<node name="InitDrive" pkg="C51_CarOperation" type="DRC_Vehicle_Init_server.py" />

<!-- Start Drive Init server -->
<node name="WayPointsDriving" pkg="C51_CarOperation" type="DRC_Vehicle_Drive_server.py" />


<!-- Start Drive Finish server -->
<node name="FinishDrive" pkg="C51_CarOperation" type="DRC_Vehicle_Finish_server.py" />

</launch>
  

